components/search/SearchForm.vue

```diff
@@ -282,6 +323,24 @@
       clearQuery() {
         this.query = '';
         this.showSearchQuery = false;
+      },
+
+      toggleSearchAndRemovePill(e) {
```

Comment:
This duplicates the entire code from the computed property `pillRemoveLinkTo`.

Replace with:
```js
      toggleSearchAndRemovePill(e) {
        e.preventDefault();
        this.toggleSearchBar();
        this.$nextTick(() => this.$router.push(this.pillRemoveLinkTo));
      }
```

Reply:
thanks for this, I have updated the PR and changed the names of the search buttons as discussed